iglo/misc/apps.py

```diff
@@ -4,3 +4,14 @@
 class MiscConfig(AppConfig):
     default_auto_field = 'django.db.models.BigAutoField'
     name = 'misc'
+    
+    def ready(self):
```

Comment:
let's not add profiling and non-product related code in this PR

Reply:
Why not. It is enabled in DEBUG only.
It is very useful for debugging. 
It required a lot of work to add it, if we don't skip it now, it would be rather hard  to add it "from my memory".

I don't see any downside given how cleanly it is separated. 
I can comment out this whole block if you want or replace "if settings.DEBUG:" by "if False" to make it dead code, but it does provide tremendous value and we should not waste it by "collectively forgetting" how to profile.

Reply:
Because if we were to rollback this change then we rollback everything